### PR TITLE
Infra automation

### PR DESCRIPTION
**Pull request by ManojKrishnaChintauri**

### CODE REVIEW

File: helm_charts/flink/values.yaml [hunk 1]
```diff
@@ -1,8 +1,10 @@
 namespace: "flink-dev"
 imagepullsecrets: ""
 dockerhub: ""
-repository: "docker.io/anandp504/sunbird-oneclick"
-image_tag: "1.0.0"
+# repository: "docker.io/anandp504/sunbird-oneclick"
+repository: "manjudr/obsrv-core-pipeline"
```
[the remark this answers is not in the document]
Comment (anandp504) [reply helm_charts/flink/values.yaml:5]: @keshavprasadms Yes. We need to create a new image and push it to Sanketika's dockerhub id. We will do it soon.

File: helm_charts/druid-cluster/templates/druid_statefulset.yaml [hunk 2]
```diff
@@ -123,9 +124,9 @@ spec:
       # Optionally specify for running broker as Deployment
       kind: Deployment
       nodeType: "broker"
-      druid.port: {{ .Values.druid_broker_port }}
+      druid.port: { { .Values.druid_broker_port } }
```
[the remark this answers is not in the document]
Comment (anandp504) [reply helm_charts/druid-cluster/templates/druid_statefulset.yaml]: Removed the whitespace in the helm templating formula.

File: helm_charts/postgresql-exporter/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,18 @@
+apiVersion: v2
```
[the remark this answers is not in the document]
Comment (ManojKrishnaChintauri) [reply helm_charts/postgresql-exporter/Chart.yaml:1]: Yes keshav, we are making changes in the exporter such as adding labels and host name etc.

[the remark this answers is not in the document]
Comment (ManojKrishnaChintauri) [reply helm_charts/postgresql-exporter/Chart.yaml:1]: Druid 
- https://github.com/Sanketika-Obsrv/obsrv-automation/blob/2d8958a6af5a4247b98791d916d004fbe1d29814/helm_charts/druid-exporter/values.yaml#L11
- https://github.com/Sanketika-Obsrv/obsrv-automation/blob/2d8958a6af5a4247b98791d916d004fbe1d29814/helm_charts/druid-exporter/values.yaml#L56

Kafka
- https://github.com/Sanketika-Obsrv/obsrv-automation/blob/2d8958a6af5a4247b98791d916d004fbe1d29814/helm_charts/kafka-exporter/values.yaml#L21-L23
- https://github.com/Sanketika-Obsrv/obsrv-automation/blob/2d8958a6af5a4247b98791d916d004fbe1d29814/helm_charts/kafka-exporter/values.yaml#L47

Postgres
- https://github.com/Sanketika-Obsrv/obsrv-automation/blob/2d8958a6af5a4247b98791d916d004fbe1d29814/helm_charts/postgresql-exporter/values.yaml#L34
- https://github.com/Sanketika-Obsrv/obsrv-automation/blob/2d8958a6af5a4247b98791d916d004fbe1d29814/helm_charts/postgresql-exporter/values.yaml#L135

**Pull request merged**